expat/xmlwf/xmlwf.c

```diff
@@ -178,7 +178,7 @@ is equivalent to lexicographically comparing based on the character number. */
 
 static int
 attcmp(const void *att1, const void *att2) {
-  return tcscmp(*(const XML_Char **)att1, *(const XML_Char **)att2);
+  return tcscmp(*(const XML_Char * const *)att1, *(const XML_Char * const *)att2);
```

Comment:
clang-format CI [asks us](https://github.com/libexpat/libexpat/actions/runs/3339947053/jobs/5529494973#step:6:119) to apply this change on top:
```diff
--- a/expat/xmlwf/xmlwf.c
+++ b/expat/xmlwf/xmlwf.c
@@ -178,7 +178,7 @@ is equivalent to lexicographically comparing based on the character number. */
 
 static int
 attcmp(const void *att1, const void *att2) {
-  return tcscmp(*(const XML_Char * const *)att1, *(const XML_Char * const *)att2);
+  return tcscmp(*(const XML_Char *const *)att1, *(const XML_Char *const *)att2);
 }
 
 static void XMLCALL
```

Reply:
Done.